pkgs/development/python-modules/pyautogui/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python3Packages
+, pkgs
+, fetchzip
+, mouseinfo
+, pygetwindow
+, pymsgbox
+, pyperclip
+, pyrect
+, pyscreeze
+, pytweening
+, tkinter
+, xlib
+}:
+buildPythonPackage rec {
+  pname = "pyautogui";
+  version = "0.9.53";
+
+  src = fetchFromGitHub {
+    owner = "asweigart";
+    repo = "pyautogui";
+    rev = "5e4acb870f2e7ce0ea1927cc5188bc2f5ab7bbbc";
+    sha256 = "sha256-R9tcTqxUaqw63FLOGFRaO/Oz6kD7V6MPHdQ8A29NdXw=";
+  };
+
+  # This package requires access to a X server for the import success
+  # You can test in a local nixpkgs tree by using:
+  # NIX_PATH=nixpkgs=$(pwd) nix-shell --run 'python3 -c "import pyautogui"' -p python3Packages.pyautogui python3
+  doCheck = false;
+
+  patches = [
+    # https://github.com/asweigart/pyautogui/issues/598
+    ./fixes.patch
```

Comment:
fixes is very generic. Please be a bit more specific.

Reply:
I've renamed to a better name